field_utils/MockField.F90

```diff
@@ -0,0 +1,170 @@
+module MockField_mod
+
+   implicit none
+
+   public :: MockField, MAXLEN
+
+   private
+
+   integer, parameter :: MAXLEN = 80
+   integer, parameter :: SUCCESS = 0
+   integer, parameter :: ERROR = SUCCESS - 1
+
+   ! Mock for ESMF_Field
+   type :: MockField
+      private
+      real(R64), allocatable :: f_(:, :)
```

Comment:
I do not see where R64 comes from.  How can this even compile?

Reply:
fixed